graph/patterns/default-properties.md

```diff
@@ -0,0 +1,201 @@
+# Default properties
+
+Microsoft Graph API Design Pattern
+
+*The default properties pattern allows API producers to omit specific properties from the response unless they are explicitly requested using `$select`. The pattern also uses an instance annotation to inform callers that other properties are also available. The same annotation is also use to encourage callers to use `$select`.*
+
+## Problem
+
+API producers want to control the set of properties that their entities return by default, when the incoming request does not specify a `$select`. This can be desirable when an entity type has many properties or an API producer needs to add properties that are computationally expensive to return by default.
```

Comment:
```suggestion
API designers want to control the set of properties that their entities return by default, when the incoming request does not specify a `$select`. This can be desirable when an entity type has many properties or an API producer needs to add properties that are computationally expensive to return by default.
```

Reply:
Will take this change.